assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -111,6 +128,11 @@ const StoriesInspectorControls = (props) => {
     }
   );
 
+  const archiveLink = `<a target="__blank" href="${archiveURL}">${__(
```

Comment:
Why is this a string and not a React component?

Should not be using `RawHTML` for this

Reply:
updated to use component instead as suggested.